### DISCUSSION TITLE
Remove redundant forum posts

### DISCUSSION THREAD

**Discussion by pinguupinguu** [RetroAchievements/RAWeb#1100] [Ideas]
Was thinking about it tonight, and realized #727 was brought up some time ago.

Wondering if a few things could be done:

- Axe the Recent Forum Activity section on the front page. It takes up more room than is really needed:
![image](https://user-images.githubusercontent.com/65916635/185773857-30aba653-fc83-45d3-8d8c-39cce887c32a.png)
- Consider removing the [Forum Post History](https://retroachievements.org/forumposthistory.php) page? I don't know how much it's needed if we have the sidebar on main index page. Not to mention that forums can be searched by (albeit not the best in terms of optimization).
![image](https://user-images.githubusercontent.com/65916635/185773915-4d771fd3-16de-45e1-a09b-94bd3f5bd996.png)

With this I think implementing the Active Forum Topics idea might work. I don't know what the framework for anything is, but I'd be interested in some thoughts.

**Comment by Jamiras** [comment]
> Consider removing the [Forum Post History](https://retroachievements.org/forumposthistory.php) page? I don't know how much it's needed if we have the sidebar on main index page.

The "more..." link goes there, as does the "Recent Posts" menu item. And it's also used for each subsequent page (`Next 25 >>`).

I do see the Forum Activity widget on the main page being redundant with the same sidebar on the forum index page. It only shows the four most recent posts, whereas the sidebar shows eight and the dedicated page shows 25.

It seems like something that doesn't really belong on the main page. People who want to participate in the forums can get to the main Forum page (or the history page) through the site menu.

I use the "Recent Posts" menu item almost exclusively. I rarely look at the smaller lists on the main page or the forum index sidebar.

**Comment by Blazekickn** [comment]
A ton of websites have something like that where the most recent topics are on the front page. I'm not usually going out of my way to access the forums, but it being on the home page still lets me know of important things i may or may not want to comment on.

**Comment by KingS1zzle** [comment]
I believe the Forum Activity widget serves a greater purpose for visibility wise. Eliminating the widget would not only reduce the amount of users who don't check the forums often, but would also reduce the amount activity within the forums. 
Having the widget on the front page allows users to glance newly posted messages and may contain their interests.
Potentially removing visibility from the forums might suggest we only communicate via the Discord.

Instead of removing, I would suggest moving around other aspects of the front page, if there is thought of how much space is needed.
I find the Currently Online widget to be better suited underneath the Statistics widget, as it was in previous versions of RA Web. 
Active Players could be moved underneath the Currently Online stats as well. Maybe widening RA Web generally would help this idea. that way RP isn't shortened while still being on the right side of the main page.

Typically a home page is majority used for quick easy convenience. Even though you can find a bunch of stuff elsewhere, the home page allows anyone to quickly open it and take a quick glance.
